sunpy/map/mapbase.py

```diff
@@ -752,15 +749,71 @@ def date(self):
             # UTC is the FITS standard default
             timesys = self.meta.get('timesys', 'UTC')
 
+        return parse_time(time, scale=timesys.lower())
+
+    @property
+    def date_beg(self):
+        """
+        Time of the beginning of the image acquisition.
+
+        Taken from the DATE-BEG FITS keyword.
+        """
+        return self._get_date('date-beg')
+
+    @property
+    def date_end(self):
+        """
+        Time of the end of the image acquisition.
+
+        Taken from the DATE-END FITS keyword.
+        """
+        return self._get_date('date-end')
+
+    @property
+    def date_avg(self):
```

Comment:
```suggestion
    def date_average(self):
```

or maybe

```suggestion
    def date_center(self):
```

Reply:
I think worth sticking with `average`, given this is the wording in FITS and this doesn't always have to be the center of the observation interval?